pkg/operator/ceph/pool/controller.go

```diff
@@ -295,9 +292,8 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 			if r.blockPoolChannels[blockPoolChannelKey].monitoringRunning {
 				logger.Debug("external rgw endpoint monitoring go routine already running!")
 			} else {
-				checker := newMirrorChecker(r.context, r.client, r.clusterInfo, request.NamespacedName, &cephBlockPool.Spec, cephBlockPool.Name)
 				r.blockPoolChannels[blockPoolChannelKey].monitoringRunning = true
-				go checker.checkMirroring(r.blockPoolChannels[blockPoolChannelKey].stopChan)
+				go checker.checkMirroring(r.blockPoolChannels[cephBlockPool.Name].stopChan)
```

Comment:
We need to use the same key here, right?
```suggestion
				go checker.checkMirroring(r.blockPoolChannels[blockPoolChannelKey].stopChan)
```

Reply:
Right. missed that while rebasing.

Reply:
CI looks good now.